diff --git a/elements/dsp/part.cc b/elements/dsp/part.cc
index 30a0aa3..11e23ad 100644
--- a/elements/dsp/part.cc
+++ b/elements/dsp/part.cc
@@ -63,7 +63,6 @@ void Part::Init(uint16_t* reverb_buffer) {
   
   for (size_t i = 0; i < kNumVoices; ++i) {
     voice_[i].Init();
-    ominous_voice_[i].Init();
   }
   
   reverb_.Init(reverb_buffer);
@@ -158,20 +157,7 @@ void Part::Process(
   // Render each voice.
   for (size_t i = 0; i < kNumVoices; ++i) {
     float midi_pitch = note_[i] + performance_state.modulation;
-    if (easter_egg_) {
-      ominous_voice_[i].Process(
-          patch_,
-          midi_pitch,
-          performance_state.strength,
-          i == active_voice_ && performance_state.gate,
-          (i == active_voice_) ? blow_in : silence_,
-          (i == active_voice_) ? strike_in : silence_,
-          raw_buffer_,
-          center_buffer_,
-          sides_buffer_,
-          size);
-    } else {
-      // Convert the MIDI pitch to a frequency.
+    // Convert the MIDI pitch to a frequency.
       int32_t pitch = static_cast<int32_t>((midi_pitch + 48.0f) * 256.0f);
       if (pitch < 0) {
         pitch = 0;
@@ -191,7 +177,6 @@ void Part::Process(
           center_buffer_,
           sides_buffer_,
           size);
-    }
     
     // Mixdown.
     for (size_t j = 0; j < size; ++j) {
@@ -204,11 +189,9 @@ void Part::Process(
   }
   
   // Pre-clipping
-  if (!easter_egg_) {
-    for (size_t i = 0; i < size; ++i) {
-      main[i] = SoftLimit(main[i]);
-      aux[i] = SoftLimit(aux[i]);
-    }
+  for (size_t i = 0; i < size; ++i) {
+    main[i] = SoftLimit(main[i]);
+    aux[i] = SoftLimit(aux[i]);
   }
   
   // Metering.
@@ -223,13 +206,8 @@ void Part::Process(
     panic_ = true;
   }
   
-  if (easter_egg_) {
-    float l = (patch_.exciter_blow_level + patch_.exciter_strike_level) * 0.5f;
-    scaled_exciter_level_ = l * (2.0f - l);
-  } else {
-    exciter_level *= 16.0f;
-    scaled_exciter_level_ = exciter_level > 0.1f ? 1.0f : exciter_level;
-  }
+  exciter_level *= 16.0f;
+  scaled_exciter_level_ = exciter_level > 0.1f ? 1.0f : exciter_level;
 
   resonator_level *= 16.0f;
   scaled_resonator_level_ = resonator_level < 1.0f ? resonator_level : 1.0f;
diff --git a/elements/dsp/part.h b/elements/dsp/part.h
index 6936812..0ed6fd9 100644
--- a/elements/dsp/part.h
+++ b/elements/dsp/part.h
@@ -32,7 +32,6 @@
 #include "stmlib/stmlib.h"
 
 #include "elements/dsp/fx/reverb.h"
-#include "elements/dsp/ominous_voice.h"
 #include "elements/dsp/patch.h"
 #include "elements/dsp/voice.h"
 
@@ -76,20 +75,15 @@ class Part {
   inline bool bypass() const { return bypass_; }
   inline void set_bypass(bool bypass) { bypass_ = bypass; }
 
-  inline bool easter_egg() const { return easter_egg_; }
-  inline void set_easter_egg(bool easter_egg) { easter_egg_ = easter_egg; }
-
   inline ResonatorModel resonator_model() const { return resonator_model_; }
   inline void set_resonator_model(ResonatorModel r) { resonator_model_ = r; }
   
  private:
   Patch patch_;
   Voice voice_[kNumVoices];
-  OminousVoice ominous_voice_[kNumVoices];
   
   bool panic_;
   bool bypass_;
-  bool easter_egg_;
   bool previous_gate_;
   float note_[kNumVoices];
   
